lib/wasabi/parser.rb

```diff
@@ -247,8 +248,7 @@ def input_output_for(operation, input_output)
         port_type_operation = @port_type_operations[binding_type][operation_name]
       end
 
-      port_type_input_output = port_type_operation &&
-        port_type_operation.element_children.find { |node| node.name == input_output }
+      port_type_input_output = port_type_operation&.element_children&.find { |node| node.name == input_output }
```

Comment:
just noting this: the lonely operator was introduced in ruby 2.3x but this gem still declares compatibility with >= 1.9.2

in practice, this is most likely an oversight, as savon projects have a de facto policy of covering only supported rubies (1.9x isnt even covered in the test matrix)